src/commcare_cloud/ansible/roles/kernel_tune/tasks/disk_scheduler.yml

```diff
@@ -0,0 +1,34 @@
+- stat:
+    path: /etc/default/grub
+  register: uses_grub
+
+- set_fact:
+      raw_disk_list: "{{ hostvars[inventory_hostname].ansible_devices.keys() | map('regex_search', 'sd.*') | select('string') | list }}"
+
+- name: set disk scheduler to noop for every raw device
+  command: "echo noop > /sys/block/{{ item }}/queue/scheduler"
+  with_items: "{{ raw_disk_list }}"
+
+- name: Add elevator=noop to default grub config if it's empty
+  lineinfile:
+    dest: /etc/default/grub
+    regexp: '^GRUB_CMDLINE_LINUX_DEFAULT=""$'
```

Comment:
i have noticed that in aws env this value is actually not empty so the transparent hugepage also hasn't been applied.

Reply:
@dannyroberts  there is a slight change required to make it applicable for AWS Env, I'll leave that decision on you.